caddy/caddymain/run.go

```diff
@@ -55,6 +56,8 @@ func init() {
 	flag.BoolVar(&plugins, "plugins", false, "List installed plugins")
 	flag.StringVar(&certmagic.Email, "email", "", "Default ACME CA account email address")
 	flag.DurationVar(&certmagic.HTTPTimeout, "catimeout", certmagic.HTTPTimeout, "Default ACME CA HTTP timeout")
+	flag.BoolVar(&jsonFrom, "jsonfrom", false, "From JSON stdin to Caddyfile stdout")
+	flag.BoolVar(&jsonTo, "jsonto", false, "From Caddyfile stdin to JSON stdout")
```

Comment:
What if we call the flag `from-json` and `to-json` ... makes much more sense in English.